<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramPageStdio</class>
 <widget class="QWidget" name="ProgramPageStdio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Standard Input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Source:</string>
            </property>
            <property name="buddy">
             <cstring>combo_stdin_redirection</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="combo_stdin_redirection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>/dev/null</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pipe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>File</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_stdin_file">
            <property name="text">
             <string>File:</string>
            </property>
            <property name="buddy">
             <cstring>combo_stdin_file</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="combo_stdin_file">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_stdin_pipe_help">
            <property name="text">
             <string>Input data can be read from a pipe (behaves similar to a terminal).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_stdin_dev_null_help">
            <property name="text">
             <string>No input data will be available.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_stdin_file_help">
            <property name="text">
             <string>Input data can be read from the specified file.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Standard Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Target:</string>
            </property>
            <property name="buddy">
             <cstring>combo_stdout_redirection</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="combo_stdout_redirection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>/dev/null</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Individual Log Files</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Continuous Log File</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_stdout_file">
            <property name="text">
             <string>File:</string>
            </property>
            <property name="buddy">
             <cstring>edit_stdout_file</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_stdout_individual_log_help">
            <property name="text">
             <string>Output data is written to an individual log file (&lt;timestamp&gt;_stdout.log) per program run.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_stdout_file_help">
            <property name="text">
             <string>Output data is written to the specified file.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_stdout_dev_null_help">
            <property name="text">
             <string>Output data is discarded.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edit_stdout_file"/>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_stdout_continuous_log_help">
            <property name="text">
             <string>Output data is written to a continuous log file (continuous_stdout.log).</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Standard Error</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Target:</string>
            </property>
            <property name="buddy">
             <cstring>combo_stderr_redirection</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_stderr_file">
            <property name="text">
             <string>File:</string>
            </property>
            <property name="buddy">
             <cstring>edit_stderr_file</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_stderr_stdout_help">
            <property name="text">
             <string>Error data is written to the same target as selected for stdout.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_stderr_dev_null_help">
            <property name="text">
             <string>Error data is discarded.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_stderr_individual_log_help">
            <property name="text">
             <string>Error data is written to an individual log file (&lt;timestamp&gt;_stderr.log) per program run.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_stderr_file_help">
            <property name="text">
             <string>Error data is written to the specified file.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edit_stderr_file"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="combo_stderr_redirection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>/dev/null</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Individual Log Files</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Continuous Log File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Standard Output</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_stderr_continuous_log_help">
            <property name="text">
             <string>Error data is written to a continuous log file (continuous_stderr.log).</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_stdin_redirection</tabstop>
  <tabstop>combo_stdin_file</tabstop>
  <tabstop>combo_stdout_redirection</tabstop>
  <tabstop>edit_stdout_file</tabstop>
  <tabstop>combo_stderr_redirection</tabstop>
  <tabstop>edit_stderr_file</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
